.buildkite/ftr_oblt_stateful_configs.yml

```diff
@@ -10,6 +10,9 @@ disabled:
   - x-pack/plugins/observability_solution/profiling/e2e/ftr_config_runner.ts
   - x-pack/plugins/observability_solution/profiling/e2e/ftr_config.ts
 
+  #FTR configs
+  - x-pack/plugins/observability_solution/uptime/e2e/config.ts
```

Comment:
@shahzad31 could you confirm if this config should be disabled and not run on kibana CI?

Reply:
it should be run, why do we want to disable it?

Reply:
It got lost a while ago and we didn't run it. I just fixed a script and it pointed the configs that are not in manifest file. It was one if them

Reply:
got it, let's hope tests aren't broken since :)

Reply:
```
[kbn-test/improve-ftr-configs-checker][~/github/kibana]$ node scripts/functional_tests.js --config x-pack/plugins/observability_solution/uptime/e2e/config.ts
 warn ❗️❗️❗️
 warn ❗️❗️❗️
 warn ❗️❗️❗️
 warn    Don't forget to use `node scripts/build_kibana_platform_plugins` to build plugins you plan on testing
 warn ❗️❗️❗️
 warn ❗️❗️❗️
 warn ❗️❗️❗️
ERROR UNHANDLED ERROR
ERROR Error: No tests defined.
          at FunctionalTestRunner.runHarness (functional_test_runner.ts:233:15)
          at FunctionalTestRunner.getTestStats (functional_test_runner.ts:153:23)
          at checkForEnabledTestsInFtrConfig (run_ftr.ts:44:27)
          at run_tests.ts:72:61
          at tooling_log.ts:84:18
          at runTests (run_tests.ts:64:5)
          at description (cli.ts:24:7)
          at run.ts:73:10
          at withProcRunner (with_proc_runner.ts:29:5)
          at run (run.ts:71:5)
```

Somehow there are no tests defined in config. I will keep it as `disabled`. If there should be some tests in config, I think code owners should wok on it separately and re-enable config

Reply:
it's executed via a custom script 

in uptime plugin, you can do 

`node scripts/e2e.js`

Reply:
ok, then it is correct to keep it under `disabled`: we enable only the configs that are run classical way through `scripts/functional_tests`